utils/flattenZoomResults.js

```diff
@@ -1,19 +1,17 @@
 function flattenZoomResults(zoomResults){
+  if (!zoomResults.length) {
+    console.log("Zoom rooms are currently empty");
+    return zoomResults;
+  }
   if (zoomResults[0].room) {
     console.log('🧚‍ running sample data');
     return zoomResults
   }
 
-  let studentsPresent = []
-  // console.log(zoomResults)
-
-  if (zoomResults.length < 1 || zoomResults === undefined) {
-    console.log("There are no students in any Zoom Room");
-    return false;
-  }
+  let studentsPresent = [];
 
   for (var i = 0; i < zoomResults.length; i++) {
-    let session = zoomResults[i].liveAttendance.map(student => {
+    let session = zoomResults[i].liveAttendance.map((student) => {
```

Comment:
parens around a lone argument are not necessary per AirBnB style guide

Reply:
I think Magee wants us to keep them in?

Reply:
That's 100% right.  I want them.  This old person with bad vision finds it easier to skim what's happening with the cues given by the parens.